sites/public/cypress/e2e/pages/listings-map.spec.ts

```diff
@@ -1,5 +1,5 @@
-// When testing this locally, ensure your screen size compares to Cypress's screen size as it will affect the way the map creates clusters
-describe.skip("Listings map", function () {
+// When testing this locally outside of a test run, ensure your screen size compares to Cypress's screen size as it will affect the way the map creates clusters
```

Comment:
Do we know what Cypress's screen size is? If so could we add it in this comment?

Reply:
Oh is it the 1500 x 800 set just below?

Reply:
Yep yep!